search/config.py

```diff
@@ -219,3 +219,11 @@
 FLASKS3_USE_HTTPS = os.environ.get('FLASKS3_USE_HTTPS', 1)
 FLASKS3_FORCE_MIMETYPE = os.environ.get('FLASKS3_FORCE_MIMETYPE', 1)
 FLASKS3_ACTIVE = os.environ.get('FLASKS3_ACTIVE', 0)
+
+# Settings for display of release information
+RELEASE_NOTES_URL = 'https://confluence.cornell.edu/x/fjs2FQ'
```

Comment:
Consider putting release notes in git repo.

Reply:
Release notes go with the tags (https://github.com/cul-it/arxiv-search/releases); mentioned in  https://confluence.cornell.edu/x/LROiF but could be made clearer. The Confluence page is intended for consumption by a broader/non-technical audience. Still experimenting with the public comms part of the release process

Reply:
The release notes are a great description for what is in this release that may be a benefit to non-technical and technical audiences. Many projects have in their source controlled documents a release log or a history of their releases. I think the release documents should be preserved in the git repo because then it will be always with the git repo, no matter what we do with confluence or github.